spring-cloud-gcp-samples/spring-cloud-gcp-data-datastore-sample/pom.xml

```diff
@@ -18,7 +18,17 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-gcp-starter-data-datastore</artifactId>
 		</dependency>
-  </dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-rest</artifactId>
+		</dependency>
+        <dependency>
```

Comment:
Spacing seems off.

Reply:
I think tabs is what Spring Team prefers, but I don't think they're too particular about it in the xml files.